.github/workflows/rspec_rubocop.yml

```diff
@@ -0,0 +1,77 @@
+# This workflow will download a prebuilt Ruby version, install dependencies and run tests with Rake
+# For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
+
+name: Ruby
+
+on: [pull_request, push]
+
+permissions:
+  contents: read
+
+env:
+  CC: ccache gcc
+  CXX: ccache g++
+  THRIFT_SHA256SUM: 2289d02de6e8db04cbbabb921aeb62bfe3098c4c83f36eec6c31194301efa10b  thrift-0.10.0.tar.gz
+  THRIFT_URL: http://archive.apache.org/dist/thrift/0.10.0/thrift-0.10.0.tar.gz
+  THRIFT_FILE: thrift-0.10.0.tar.gz
+  THRIFT_DIR: thrift-0.10.0
+  DEBIAN_FRONTEND: noninteractive # for apt to be noninteractive
+  ImageOS: ubuntu20
```

Comment:
This is required for running `setup-ruby` on [self-hosted runners](https://github.com/ruby/setup-ruby#using-self-hosted-runners).